presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -306,6 +306,14 @@ Accessors
 
     Returns the great-circle distance between two points on Earth's surface in kilometers.
 
+Aggregations
+------------
+.. function:: convex_hull_agg(Geometry) -> Geometry
+
+    Returns a geometry that is the convex hull of all the geometries in the collection/all input geometries.
```

Comment:
This sentence needs to be removed. It contains the same information as the next one.

Reply:
Done